src/components/fields/ArrayField.js

```diff
@@ -378,6 +383,17 @@ class ArrayField extends Component {
       </div>
     );
   }
+
+  renderHidden() {
+    const {idSchema} = this.props;
+    const {items} = this.state;
+    return (
+      <HiddenWidget
+        id={idSchema && idSchema.$id}
+        value={items}
+      />
+    );
+  }
```

Comment:
This was the easiest way to render the HiddenWidget instead of the array one. But it's done a bit differently than the other widgets (which are declared in the `altWidgetMap` in utils right?).